### PR TITLE
Long policy pre-notification

### PR DESCRIPTION
**Pull request by peterlangmar**
- understanding a long text on the spot at the meeting is not realistic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,9 @@ Purpose: Coordinate event organizing
 3. You can amend and clarify your proposal, integrate objections by adding changes to the pull request. The time window for Clarifying Questions, Reactions and Objections restart with each amendment or clarification.
 4. Merge the pull request if there are no objections in the given period.
 
+### Policy: Long policies
+Policy proposals longer than 10 sentences should be sent out on Slack to raise attention to enable the circle to prepare properly for the governance meeting. If the policy if complex than pre-governance discussions are preferable.
```
Comment (cassus) [inline README.md]: This already happens automatically when you create the pull req. The github bot sends it to slack, people subscribed to the repo get emails. So in the current form I see no use of this policy.

Comment (cassus) [inline README.md]: Maybe we need
1. some time before the governance meeting to review the change
2. @-notifying affected ppl to raise awareness (either on slack or github)

**Comment by peterlangmar** [comment]
@cassus fixed thanks

**Comment by cassus** [comment]
:+1:

**Comment by nagydan1** [comment]
:+1:

**Pull request merged**